api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -35,6 +35,7 @@ message AccessLog {
   // #. "envoy.access_loggers.file"
   // #. "envoy.access_loggers.http_grpc"
   // #. "envoy.access_loggers.tcp_grpc"
+  // #. "envoy.access_loggers.open_telemetry"
```

Comment:
@phlax I think this is another extension point which actually does not lookup by type_url. It must do this conversion internally. However, it should still be covered by your new system. Can you take a look at this and backfill it? Once this is done we will no longer need to list these out (I think @phlax is going through and deleting the manual ones where present).

cc @htuch @adisuissa since I'm not sure if we should be moving everything over type TypedExtensionConfig? This is one of our earliest extension points and obviously was never migrated.

Reply:
i think it should work the same as the other accesslog protos

for some reason the docs are not building correctly - its not including the generated `.rst` snippet that it generates for this extension https://dev.azure.com/cncf/envoy/_build/results?buildId=67403&view=logs&j=91e633a5-4907-5da1-5862-92a79fe2387a&t=f29e24e7-a093-5724-80bd-d79975d268cb&l=2714

its not obvious to me why its failing on that

Reply:
When I run it locally (ci/run_envoy_docker.sh 'ci/do_ci.sh docs') I get the following warning:
"/source/generated/rst/api-v3/extensions/access_loggers/open_telemetry/v3alpha/logs_service.proto.rst: WARNING: document isn't included in any toctree"
Could that be the issue? If so, what does it mean?

Reply:
Oh I see that this is exactly what @phlax pointed out.
@yanavlasov @htuch do you know why this happens?

Reply:
Never mind, I think I fixed it - only */v3/* paths were covered for access logger docs, so I added v3alpha too.

Reply:
yep - cool - once the docs have built we can see the linking etc

Reply:
the access log page is here:

https://storage.googleapis.com/envoy-pr/48325a4/docs/api-v3/config/accesslog/v3/accesslog.proto.html#config-accesslog-v3-accesslog

and the extension page is here:

https://storage.googleapis.com/envoy-pr/48325a4/docs/api-v3/extensions/access_loggers/open_telemetry/v3alpha/logs_service.proto.html#extensions-access-loggers-open-telemetry-v3alpha-opentelemetryaccesslogconfig

Reply:
Looks like all the links work now, thanks!

Reply:
So can we now just add the extension category link and remove the manual list?

Reply:
the extension category link is added automatically

there is a ticket to address how best to show/update the rest here #15191

Reply:
OK sounds good, seems like we can just delete them now but up to you.

Reply:
Done (by merge)